dfastmi/batch/AnalyserAndReporterWaqua.py

```diff
@@ -68,22 +64,16 @@ def analyse_and_report_waqua(
     reduced_output : bool
         Flag to indicate whether WAQUA output should be reduced to the area of
         interest only.
-    reach : str
-        Name of the reach.
-    q_location : str
-        Name of the location at which the discharge is
     tstag : float
         Number of days that the river is stagnant.
-    Q : Vector
+    discharges : Vector
```

Comment:
Maybe reference Q in the comment here?

Reply:
How do you exactly mean?
e.g. previously known as Q

Reply:
(is Q array or discharges), similar this for T/fraction_of_year

Reply:
done